wordle-web/src/components/currentUser.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-btn @click="showDialog = true">
+    {{ Player.Name.value }}
+  </v-btn>
+  <v-dialog :model-value="showDialog" @update:model-value="close" persistent>
+    <v-card>
+      <v-card-text>
+        Set or change your username:
+        <v-text-field v-model="newName" :placeholder="Player.Name.value" />
+      </v-card-text>
+      <v-card-actions>
+        <v-btn @click="close">Cancel</v-btn>
+        <v-btn @click="confirm">Confirm</v-btn>
```

Comment:
Typically, with material design, these are right justified 

You could add a `<v-spacer /> above the buttons and that should take care of it!